benchmarks/CMakeLists.txt

```diff
@@ -8,33 +8,34 @@ macro (do_benchmark source)
 	add_executable (benchmark_${source} ${SOURCES})
 	add_dependencies (benchmark_${source} kdberrors_generated)
 
-	target_link_elektra(benchmark_${source} elektra-kdb elektra-meta)
+	target_link_elektra (benchmark_${source} elektra-kdb elektra-meta)
 
-	set_target_properties (benchmark_${source} PROPERTIES
-			COMPILE_DEFINITIONS HAVE_KDBCONFIG_H)
+	set_target_properties (benchmark_${source} PROPERTIES COMPILE_DEFINITIONS HAVE_KDBCONFIG_H)
 
 	add_custom_target (benchmark_${source}_callgrind
-		rm -f *.out
-		COMMAND valgrind --tool=callgrind --callgrind-out-file=callgrind.out "$<TARGET_FILE:benchmark_${source}>" > output.out
-		# COMMAND strace -o strace.out "${command}"
```

Comment:
Where is the comment? I thought this is *only* reformatting?

Reply:
Unfortunately `cmake-format` currently [moves comments to the previous line](https://github.com/cheshirekow/cmake_format/issues/33), if there is no newline before the comment. Inside CMake commands even a newline before the comment will not protect it against this unfortunate reformatting.  

For example, the formatted output of this code:

```cmake
add_custom_target (benchmark_${source}_callgrind
       COMMAND rm -f *.out 

       # COMMAND strace -o strace.out "${command}"
       COMMAND valgrind --tool=callgrind
         --callgrind-out-file=callgrind.out "$<TARGET_FILE:benchmark_${source}>" > output.out
       WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
```
will look like this:

```cmake
add_custom_target (benchmark_${source}_callgrind
       COMMAND rm -f *.out # COMMAND strace -o strace.out "${command}"
       COMMAND valgrind --tool=callgrind
               --callgrind-out-file=callgrind.out "$<TARGET_FILE:benchmark_${source}>" > output.out
       WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}")
```

. That is the reason why I removed the comment. I just assumed it was not that important, and that we could always restore it via `git` anyway.